concent-builder/configure-and-run-tests.yml

```diff
@@ -0,0 +1,55 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Copy concent and virtualmachine for deployment build
+          command: cp                                                                  \
+              --recursive                                                              \
+              "{{ build_dir }}/concent-deployment/containers/build/{{ item.path }}"    \
```

Comment:
Isn't there some kind of setup step in the pipeline where we could put this? I think this setup should not be a part of the step that runs the tests.

Reply:
This task depends on the `build-test-and-push` playbook, so we can add it there. However I think that it's in the right place. The playbook creates independent environment that copies a specific concent version with a specific version of the tests, launches them and then deletes them.

Reply:
The problem is that if this initialization fails, it's counted as a failure of the code being tested. I'd prefer to have separate initialize/test/cleanup steps. May be separate playbooks if there's no other way.

Reply:
The `cleanup` step must depends on `test`step. When the `test` playbook fails, the `cleanup` playbook will not be executed. I maybe find solution how to execute always playbook in jenkins but I think we should keep this in the one playbook.

Reply:
Well, OK let's keep it like this for now but I'd like to find a solution to make it more modular in the future.